tests/e2e/conftest.py

```diff
@@ -418,3 +425,209 @@ def mcg_system_recovery_check(mcg_sys_setup_dict):
         )
 
     return mcg_system_recovery_check
+
+
+@pytest.fixture(scope="class")
+def jenkins_factory(request):
+    return jenkins_factory_fixture(request)
+
+
+def jenkins_factory_fixture(request):
```

Comment:
we already have fixtures for workloads with these names, is it possible to combine these two to  a single fixture?

Reply:
done